static/src/js/components/Page/Page.scss

```diff
@@ -1,4 +1,29 @@
 .page {
+  &--has-background-image {
+    position: relative;
+    background: url('@/assets/images/yellow-sea-swirls.jpg');
+    background-repeat: no-repeat;
+    background-size: auto 20rem;
```

Comment:
could this be a bit larger? The "Manage NASA Common Metadata Repository metadata with the Metadata Management Tool" pushes the two dialog boxes down.

Reply:
We wouldnt be able to make it bigger without losing quality in the image. We might be able to circle back later and find a better/bigger image if we wanted to revamp the layout. I tried to layer in some super quick improvements using the old imagery for now.